app/models/notifications/also_commented.rb

```diff
@@ -18,7 +18,11 @@ def self.notify(comment, _recipient_user_ids)
       recipient_ids = commentable.participants.local.where.not(id: [commentable.author_id, actor.id]).pluck(:owner_id)
 
       User.where(id: recipient_ids).find_each do |recipient|
-        next if recipient.is_shareable_hidden?(commentable)
+        if recipient.is_shareable_hidden?(commentable) ||
+          Notifications::MentionedInComment.where(target: comment.mentions.where(person: recipient.person)).exists?
```

Comment:
Maybe you can also create a `Commented` module like the `Mentioned` with common logic, because `deleted_translation_key` is also the same for both. Otherwise I have no problem if it isn't a one-liner, I still think the method-name `mention_notification_exists?` is easier to read/understand than the AR-query and that was my main goal here. But if it is only used for commented-notifications it doesn't belong to `Notification`.